Recursion/Print_Decrease_Increase_of_a_number.cpp

```diff
@@ -0,0 +1,51 @@
+// Write a program that will print a number first in decrease order then in increase order using recursion.
+#include <iostream>
+class PrintDecreaseIncrease{
+    private:
+        int n;
+        //Recursive function 
+        void pdi(int n){
+            if(n==0) return;
+            std::cout<<n<<std::endl;
+            pdi(n-1);
+            std::cout<<n<<std::endl;
+        }
+    public:
+        // constructor 
+        PrintDecreaseIncrease(int x): n(x){}
+        void printDI(){
+            std::cout<<"The ouput is:"<<std::endl;
+            pdi(n);
+        }
+};
+int main() {
+    int n;
+    std::cout<<"Enter a number:";
+    std::cin>>n;
+    PrintDecreaseIncrease myPdi(n);
+    myPdi.printDI();
+    return 0;
```

Comment:
In real inteview using the proper variable names would help you having edge over other ones. One example.

``
int inputNumber;
std::cout << "Enter a number: ";
std::cin >> inputNumber;

``